src/babel/utils.ts

```diff
@@ -1,16 +1,24 @@
 import { types } from '@babel/core'
 
+export interface PluginOptions {
+  customAtomNames?: string[]
+}
+
 export function isAtom(
   t: typeof types,
-  callee: babel.types.Expression | babel.types.V8IntrinsicIdentifier
+  callee: babel.types.Expression | babel.types.V8IntrinsicIdentifier,
+  customAtomNames: PluginOptions['customAtomNames'] | undefined = []
```

Comment:
Can be either,
```suggestion
  customAtomNames: PluginOptions['customAtomNames'] = []
```
or
```suggestion
  customAtomNames?: PluginOptions['customAtomNames']
```
?